packages/ui-signer/src/Unlock.tsx

```diff
@@ -36,18 +37,19 @@ class Unlock extends React.PureComponent<Props, State> {
     const isLocked = pair.isLocked();
 
     return {
+      error: error,
       isError: !!error,
       isLocked,
       pair
     };
   }
 
   render () {
-    const { autoFocus, onChange, onKeyDown, password, t, tabIndex } = this.props;
-    const { isError, isLocked } = this.state;
+    const { autoFocus, label, onChange, onKeyDown, password, t, tabIndex } = this.props;
+    const { error, isError, isLocked, pair } = this.state;
 
-    if (!isLocked) {
-      return null;
+    if (pair && !isLocked) {
+      pair.lock();
```

Comment:
Ok, now we are forcing the unlocking each and every time - refrain from doing this until we have https://github.com/polkadot-js/apps/issues/190 in (and then this happens _only_ when the strategy is to unlock per transaction)